CMakeLists.txt

```diff
@@ -53,6 +53,8 @@ set(SYSLOG_NG_PACKAGE_NAME "${CMAKE_PROJECT_NAME}")
 set(SYSLOG_NG_PATH_XSDDIR "\${datadir}/syslog-ng/xsd")
 set(SYSLOG_NG_JAVA_MODULE_PATH "\${exec_prefix}/lib/syslog-ng/java-modules")
 set(SYSLOG_NG_PYTHON_MODULE_DIR "\${exec_prefix}/lib/syslog-ng/python")
+set(SYSLOG_NG_PYTHON_SYSCONF_MODULE_DIR "\${prefix}/etc/python")
+set(SYSLOG_NG_PYTHON_VENV_DIR "\${localstatedir}/python-venv")
```

Comment:
I think this should be `syslog-ng` specific, as the `${localstatedir}` (`/var`, `/usr/local/var`) can contain files and dirs from other projects/packages. If you agree, kindly update it here, at `configure.ac` and the newsfile entry, too. Thanks!

Reply:
This is syslog-ng specific, normally, that's why it is `@localstatedir@` and not stateside

If you look at our packaging this is pointing to /var/lib/syslog-ng

Reply:
Thanks, I missed that.